### PR TITLE
Cleaned up codebase of Overlays

### LINKED ISSUE #941: Cleanup codebase of Overlays
@vysakh-m _a

### PR DESCRIPTION
**Pull request by vysakh-m**
Fixes #941 

- Cleaned up `Alert`, `Modal`, `Pane`, `Toastr` & `Tooltip`

### CODE REVIEW

**Comment by vysakh-m** [comment]
@amaldinesh7 _a Please review. Also, can [this](https://github.com/bigbinary/neeto-ui/blob/e844ba787774116042f5db20de149f31982198eb/lib/components/Tooltip.js#L11) prop be removed?

**Comment by amaldinesh7** [comment]
> this

No, not in this PR. We are releasing a beta release soon, so along with that, we can do. Please do note down that this change has to be made.

File: lib/components/Pane/index.js [hunk 1]
```diff
@@ -102,108 +107,48 @@ const Pane = ({
...
+   * To specify the size of the Pane.
+   */
+  size: PropTypes.oneOf(SIZES),
+  /**
+   * To specify whether the Pane component is open or not.
    */
   isOpen: PropTypes.bool,
   /**
-   * To specify the callback which will be invoked when the pane is closed.
+   * To specify the callback which will be invoked when the Pane is closed.
```
Comment (amaldinesh7) [inline lib/components/Pane/index.js:119]:
```suggestion
   * To specify the callback which will be invoked when the close button of Pane is clicked.
```

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@vysakh-m _a Please check the comments.

**Comment by vysakh-m** [comment]
@amaldinesh7 _a Resolved suggestion.

**Review by amaldinesh7 — APPROVED** [review]

**Pull request merged**